apps/docs/pages/guides/auth/phone-login/twilio.mdx

```diff
@@ -347,8 +347,30 @@ let { data, error } = await supabase.auth.verifyOtp({
 })
 ```
 
+## Twilio Verify
+
+Twilio Verify is an alternative service by Twilio which can replace Twilio Programmable Messaging (previous described). Developers may migrate from Twilio Messaging to:
+
+1. Make use of built in fraud detection
+2. Utilize Message Services instead of a registered phone number.
+
+Twilio Verify is currently [set up as a Phone Provider](/docs/img/guides/auth-twilio/twilio-verify.png)
```

Comment:
```suggestion
Twilio Verify is available as a separate phone provider. It can't be used together with other providers, including Twilio Programmable Messaging.
```

Link is broken?